### ISSUE TITLE
on windows, jetty lock static content files.

### ISSUE THREAD

**Issue by nazoking** [gitbucket/gitbucket#1041]
when run server and restart it, sbt say

```
[error] (*:webappPrepare) java.io.FileNotFoundException: C:\Users\nazoking\git\gitbucket\target\webapp\assets\common\css\gitbucket.css (要求された操作はユーザー マップ セクションで開いたファイルでは実行できません。)
[error] Total time: 3 s, completed Jan 12, 2016 5:34:32 PM
```

「ユーザー マップ セクションで開いたファイルには要求された操作を実行できません」is 'The requested operation cannot be performed on a file with a user-mapped section open' on english.

Maybe, it relate http://www.eclipse.org/jetty/documentation/current/troubleshooting-locked-files-on-windows.html

This problem is since xsbt-web-plugin version 1.0. ( fixed on https://github.com/earldouglas/xsbt-web-plugin/issues/156 version 0.9, but it code is removed after plugin rewred on version 1.0... ).
gitbucket changed container-plugin from scalatra-sbt 0.3.5 ( that used xsbt-web-plugin 0.7.0 ) to xsbt-web-plugin 2.1.0

my solution is add this to `src/main/webapp/WEB-INF/web.xml`

```
<context-param>
    <param-name>org.eclipse.jetty.servlet.Default.useFileMappedBuffer</param-name>
    <param-value>false</param-value>
</context-param>
```

but it effect not only windows, and not only under developing ...

**Comment by nazoking** [comment]
refs #1029

**Comment by takezoe** [comment]
@nazoking Should we feedback this issue to xsbt-web-plugin?

**Comment by nazoking** [comment]
right. I reported https://github.com/earldouglas/xsbt-web-plugin/issues/297
